index.html

```diff
@@ -0,0 +1,271 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>pollstar project</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!-- the main page -->
+    <section class="home-section">
+      <header class="main-header">
+        <div class="burger">
+          <span class="bars"></span>
+          <span class="bars"></span>
+          <span class="bars"></span>
+        </div>
+        <div class="logo">
+          <a href="#"><img src="Images/logo1.png" alt="A pollstar logo" /></a>
+        </div>
+        <nav class="nav">
+          <ul class="nav-ul">
+            <a class="navlink" href="index.html"
+              ><li class="home-li">Home</li></a
+            >
+            <a class="navlink" href="./about-page/about.html"><li>About</li></a>
+            <a class="navlink" href="#"><li>News</li></a>
+            <a class="navlink" href="#"><li>Data</li></a>
+            <a class="navlink" href="#"><li>Store</li></a>
+            <li class="pallstar">Pollstar</li>
+          </ul>
+        </nav>
+      </header>
+      <main class="main">
+        <h3 class="greeting">"Hello! Sharing world"</h3>
+        <h1 class="main-tittle">The Voice of Live</h1>
+        <p class="main-p">
+          For over 40 years, Pollstar has provided the most up-to-date,
+          relevant, and useful data available for the global concert industry.
+          We deliver an unmatched archive of live entertainment data, a focus on
+          editorial excellence, and the most powerful metrics of entertainment
+          industry success available.
+        </p>
+        <h2 class="availability">We are working 24/7</h2>
+        <p class="info-p">
+          Feel free to check on us to get news of any concert in the US
+        </p>
```

Comment:
- [ ] Good job so far, but I think you miss some text in this section and it would be better if you add it 🙃 

Your design
<img width="977" alt="Screen Shot 2023-01-03 at 23 47 51" src="https://user-images.githubusercontent.com/74501523/210440686-9ff9b38a-d1cb-4afd-b934-dde2920a77a9.png">

Given design
<img width="977" alt="Screen Shot 2023-01-03 at 23 48 08" src="https://user-images.githubusercontent.com/74501523/210440710-9d8b6785-e199-47ac-b9f5-97152f7344de.png">

- [ ] Also. please make sure your design stick as much as possible to the given design like background image, and color,... You can use this [image](https://user-images.githubusercontent.com/74501523/210439976-77276323-0eb0-4c9f-80ff-f14b3905be7f.png) for the background image of this section 😉

Reply:
Fixed.